src/actions/account.js

```diff
@@ -229,27 +229,22 @@ export const login = ({ passphrase, publicKey, hwInfo }) => async (dispatch, get
   }).then(async (accountData) => {
     const expireTime = (passphrase && getState().settings.autoLog) ?
       Date.now() + accountConfig.lockDuration : 0;
+    let btcAccountData;
+    if (getState().settings.token.list.BTC) {
+      await getAccount({
+        token: tokenMap.BTC.key, networkConfig, passphrase,
+      }).then((response) => { btcAccountData = response; });
+    }
```

Comment:
IMO, this would look better, as there would be no mutation:
```js
const btcAccountData = getState().settings.token.list.BTC && await getAccount(...);
```
And in case the token is disabled, the value would just be false.